supertokens_python/recipe/multitenancy/recipe.py

```diff
@@ -216,6 +216,9 @@ async def login_methods_get(
             tenant_id, user_context
         )
```

Comment:
why do you have two api imple for multi tenancy? One here, and one in supertokens_python/recipe/multitenancy/api/implementation.py

Reply:
Removed the redundant one from `recipe.py`